client/components/diff-window.js

```diff
@@ -304,6 +320,12 @@ const DiffWindow = (props) => {
   const compare = () => {
 
     const hasVisibleChanges = hasContentChanges(before, props.value, props.type);
+    let removed = changes.removed;
+    let added = changes.added;
+    if (props.type === 'radio') {
+      removed = { added: false };
+      added = { added: true };
+    }
```

Comment:
compare function has only render at the moment, I am thinking maybe its good to move above to renderDiff !

also, I don't see `removed = { added: false };` used anywhere, am I missing something !?

Reply:
It is too late for renderdiff to determine if it is added or removed. Existing comparison for checkbox does it here. Removed is passed in previous window (line 347)

Reply:
just a thought, i think you could remove all this and do everything in radioDiff function you created .. would this work? ( I didn't test it, I just typed quickly)

`        const radioDiff = (parts) => {
      const newParts = { ...parts, removed: { added: false }, added: { added: true }}
      return (
        <p>
          {
            value === undefined ? (
              <em>{ DEFAULT_LABEL }</em>
            ) : (
              <span className={`diff ${newParts.added ? 'added' : 'removed'}`}> { value ? 'Yes' : 'No'}</span>
            )
          }
        </p>
      );
    };`

Reply:
No it will not work..